### ISSUE TITLE
accept pathnames behavior

### ISSUE THREAD

**Issue by admich** [McCLIM/McCLIM#1130]
`(accept 'pathname)` automatically complete the input string with an existing file if the completion is unique. The completion is made without any input key press from the user (like TAB). 

Is it the correct behavior? I find this uncomfortable: it is impossible to input a not existing pathname if there is an existing file with the same initial part.

**Comment by dkochmanski** [comment]
I agree that it seems fishy. no completion should happen before tab is pressed
